### ISSUE TITLE
JS error when marking text with mixed alignment in 1.0

### ISSUE THREAD

**Issue by crisbeto** [quilljs/quill#815]
When marking two paragraphs with a mixed alignment (e.g. right and center), Quill throws an error.

**Steps for Reproduction**

1. Go to the 1.0 homepage.
2. Clear the editor and paste a couple of paragraphs of text (note that apparently there can't be an empty paragraph between them).
3. Press ctrl+a to mark everything. At this point the following error is logged:

```
quill.js:9041 Uncaught TypeError: Cannot read property 'innerHTML' of nullselectItem @ quill.js:9041update @ quill.js:8907(anonymous function) @ quill.js:9543update @ quill.js:9542emit @ quill.js:4181emit @ quill.js:4002update @ quill.js:5585
```

**Extra**: I couldn't really paste over the text with the formatting to GitHub, but here's what the text looks like: 

![cxu60vl](https://cloud.githubusercontent.com/assets/4450522/17100399/c70ab64e-5276-11e6-9b71-2c76ec23593b.png)

**Platforms**: Chrome 52 on Windows 10

**Version**: 1.0.0-beta.9

**Comment by jhchen** [comment]
What application are you pasting from? I'm not able to reproduce.

**Comment by crisbeto** [comment]
[I'm pasting it from here.](http://lipsum.com/feed/html). Also here's a recording of the steps that I'm taking:

![a](https://cloud.githubusercontent.com/assets/4450522/17115557/896eca64-52bc-11e6-93b0-cb2a96dee51c.gif)

**Comment by benbro** [comment]
Maybe you can paste the delta to help reproducing the issue:
```javascript
console.log(JSON.stringify(quill.getContents()));
```

**Comment by jackmu95** [comment]
**Here the delta to reproduce it:**
```
{"ops":[{"insert":"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet."},{"attributes":{"align":"right"},"insert":"\n"},{"insert":"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet."},{"attributes":{"align":"center"},"insert":"\n"}]}
```

And here is the fix aa26d1d972dabca6acd9c31f71e87e6dde1d55ef.

**Comment by jhchen** [comment]
Thanks for the video @crisbeto I was able to reproduce from that example. @jackmu95 Thanks for your suggested commit but the issue is the picker still needs to revert back the the default option.
